cinema/views.py

```diff
@@ -34,6 +39,24 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    def get_queryset(self):
+        actors = self.request.query_params.get("actors")
+        genres = self.request.query_params.get("genres")
+        title = self.request.query_params.get("title")
+
+        queryset = self.queryset
+
+        if actors:
+            actors_ids = [int(str_id) for str_id in actors.split(",")]
```

Comment:
create staticmethod for this to avoid duplication

Reply:
thanks!